core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -4233,6 +4233,40 @@ private void checkLiteral2(String expression, String expected) {
     });
   }
 
+  @Test public void testCurrentUser() {
+    String query = "select CURRENT_USER from \"product\" where \"product_id\" = 1";
```

Comment:
you can simply write  select CURRENT_USER without from clause here

Reply:
done.